config.default

```diff
@@ -47,6 +47,10 @@ seaIceStreamsFileName = streams.cice
 # names of ocean and sea ice meshes (e.g. EC60to30, QU240, RRS30to10, etc.)
 mpasMeshName = mesh
 
+# this is the maximum number of files, beyond which xarray will close files 
+# after data has been read to prevent errors related to too many open files.
+autocloseFileCount = 1024
```

Comment:
It looks like this is the python equivalent:
https://docs.python.org/2/library/resource.html
So the code would be
```
resource.getrlimit(resource.RLIMIT_NOFILE)[0]
```